lib/importers/google/sessions/saver.rb

```diff
@@ -0,0 +1,29 @@
+module Importers
+  module Google
+    module Sessions
+      class Saver
+        def initialize(records:)
+          @records = records
+        end
+
+        def save
+          ActiveRecord::Base.transaction do
+            appointment_summaries!
+          end
+        end
+
+        private
+
+        def appointment_summaries!
+          @records.each do |record|
+            summary = AppointmentSummary.find_or_initialize_by(record.unique_identifier)
+
+            next if summary.transactions > record.transactions
```

Comment:
This was added due to an initial limitation of the data.

i.e. we got the last 90 days grouped by month meaning we would have some partial months.

This limitation has been removed and we will now only get whole months so this line has been removed.